src/components/RadioGroup/index.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react'
+import { commonLabelClasses, commonIconClasses, inputClasses, disabledClasses } from '../Checkbox'
+
+const radioClasses = 'mb2 mh2 eh-radio'
+const radioIconClasses = `${commonIconClasses} br-pill eh-radio-icon`
+
+class RadioButtonGroup extends Component {
+  static propTypes = {
+    optionList: React.PropTypes.array.isRequired
```

Comment:
Since this requires a specific shape for the object keys, would be nice to make this prop type more specific than `array`; here is an example of how I did this with the `Tab` comonent `groups` prop type:
https://github.com/eggheadio/egghead-io-styleguide/blob/master/src/components/Tabs/index.js#L14-L17